### PR TITLE
[ALMOST READY] Blueshift Gauntlets 2: Electric Boogaloo

### PR DESCRIPTION
**Pull request by gonenoculer5**
## About The Pull Request
Introducing a new method of surgery: _**The Blueshift Gauntlets!**_

These mythical bluespace tech gloves allow for a surgeon (or paramedic!) to preform effective surgeries on the fly or in the field, allowing for emergency surgery on a table or other surface in the case of illnesses such as terminal appendicitis, heart attacks, or just the casual cavity implant.

**_For the moment, this is all it will be capable of, unless I end up adding more functions to it before this releases._**

## Why It's Good For The Game

Personally, I have always thought it would be interesting to see paramedics have more tools to use in their arsenal that would facilitate less ghetto field surgery.

## Credits:

**Code by:**
Gonenoculer5
**Spriting done by:**
Papaporo Paprito

## Changelog
:cl:
add: The Blueshift Gauntlets, a pair of hyper-advanced gloves capable of manipulating bluespace folds to speed up and enhance surgery in the field.
tweak: Modified many surgery dm's to permit the usage of the BSG's as a surgery tool.
/:cl:

### CODE REVIEW

**Comment by gonenoculer5** [comment]
proper version of pull request https://github.com/TheSwain/Fulpstation/pull/384

**Comment by gonenoculer5** [comment]
Trello Card: https://trello.com/c/4SQYsWDE/6-blue-shift-gauntlets

File: code/FulpstationCode/blueshift_gauntlets/bsg.dm [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+/datum/action/summon_implements/Trigger()
+	if(owner.incapacitated())
+		to_chat(owner, "<span class='warning'>You can't use [name] while you're incapacitated!</span>")
+		return
+	if(isPhased == FALSE)
+		to_chat(owner, "<span class ='warning'>You can't use [name] while the gauntlets arent phased!</span>")
+		owner.visible_message("[owner.name] tries to summon a tool while the gauntlets are off with a dull beep.")
+		playsound(owner,'sound/machines/buzz-two.ogg',1,TRUE)
+	else
+		to_chat(owner, "<span class ='notice'>You deactivate the gauntlets, removing them from the bluespace fold!</span>")
```
Comment (ATH1909) [inline code/FulpstationCode/blueshift_gauntlets/bsg.dm]: ?

File: code/FulpstationCode/blueshift_gauntlets/bsg.dm [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	button_icon_state = "neckchop" //placeholder
+
+/datum/action/set_phase/Trigger() //Toggle the gloves phase between on or off.
+	if(owner.incapacitated())
+		to_chat(owner, "<span class='warning'>You can't use [name] while you're incapacitated!</span>")
+		return
+	if(isPhased == FALSE)
+		to_chat(owner, "<span class ='notice'>You activate the gauntlets, preparing them to phase!</span>")
+		isPhased = TRUE
+		owner.visible_message("[owner.name] activates their Blueshift Gauntlets with a low hum.")
```
Comment (ATH1909) [inline code/FulpstationCode/blueshift_gauntlets/bsg.dm]: you could use the sound effect for turning a stasis bed on (NOT the sound effect for BUCKLING someone to a stasis bed) here for the humming noise

Comment (gonenoculer5) [reply code/FulpstationCode/blueshift_gauntlets/bsg.dm]: Yeah! I was considering using that, there are a few other candidates but that one i like.

File: code/FulpstationCode/blueshift_gauntlets/bsg.dm (hunk 1 above)
Comment (gonenoculer5) [reply code/FulpstationCode/blueshift_gauntlets/bsg.dm]: huh?

Comment (gonenoculer5) [reply code/FulpstationCode/blueshift_gauntlets/bsg.dm]: the gauntlets turn themselves off if you take them off while they're on is what that does essentially, prevents the var getting stuck turned on after being removed or other werid scenarios like that by resetting the var

**Comment by gonenoculer5** [comment]
@remove32 I dont know how to resolve merge conflicts, mind lending me a hand here?

**Comment by gonenoculer5** [comment]
Currently waiting on sprites from Papaporo to release this, but other then that its done!

**Comment by gonenoculer5** [comment]
Fixed merge conflicts

**Comment by gonenoculer5** [comment]
Waited several weeks for sprites, no sprites, going to just migrate this to my personal CB.

**Pull request closed without merging**